Formula/phpmd.rb

```diff
@@ -0,0 +1,28 @@
+class Phpmd < Formula
+  desc "PHP Mess Detector"
+  homepage "https://phpmd.org"
+  url "https://static.phpmd.org/php/2.6.0/phpmd.phar"
+  sha256 "69bec1176370a3bcbb81e1d422253f70305432ecf5b2c50d04ec33adb0e20f7a"
+
+  bottle :unneeded
+
+  depends_on "php"
```

Comment:
@SMillerDev since Macos ships with php and phpmd only requires php 5.3.9 would it not be possible for us to skip the dependency here ?

Reply:
Good catch, I didn't think of that.

Reply:
@JoyceBabu can you please remove the php dependency since what ships with MacOS is good enough to support this ?